lib/ansible/modules/network/nxos/nxos_hsrp.py

```diff
@@ -272,8 +249,9 @@ def get_hsrp_group(group, interface, module):
 
 def get_commands_remove_hsrp(group, interface):
     commands = []
-    commands.append('interface {0}'.format(interface))
-    commands.append('no hsrp {0}'.format(group))
+    cmds = 'interface {0}'.format(interface), 'no hsrp {0}'.format(group)
```

Comment:
Why can't we just return this? For that matter, what purpose does this method serve?